test/postman/inference_data.json

```diff
@@ -1,120 +1,360 @@
+        "model_name":"densenet161",
+        "worker":1,
+        "synchronous":"true",
+        "file":"../examples/image_classifier/kitten.jpg",
+        "content-type":"application/json",
+        "validator":"image_classification",
+        "expected":[
+            {
+                "tiger_cat":0.4693359136581421
+            },
+            {
+                "tabby":0.4633873701095581
+            },
+            {
+                "Egyptian_cat":0.06456154584884644
+            },
+            {
+                "lynx":0.001282821292988956
+            },
+            {
+                "plastic_bag":0.00023323031200561672
+            }
+        ],
+        "tolerance":1
+    },
+    {
+        "url":"https://torchserve.s3.amazonaws.com/mar_files/alexnet.mar",
+        "model_name":"alexnet",
+        "worker":1,
+        "synchronous":"true",
+        "file":"../examples/image_classifier/kitten.jpg",
+        "content-type":"application/json",
+        "validator":"image_classification",
+        "expected":[
+            {
+                "tabby":0.31839194893836975
+            },
+            {
+                "tiger_cat":0.2651394307613373
+            },
+            {
+                "Egyptian_cat":0.24323278665542603
+            },
+            {
+                "lynx":0.16119907796382904
+            },
+            {
+                "tiger":0.006510233972221613
+            }
+        ],
+        "tolerance":1
+    },
+    {
+        "url":"https://torchserve.s3.amazonaws.com/mar_files/resnet-18.mar",
+        "model_name":"resnet-18",
+        "worker":1,
+        "synchronous":"true",
+        "file":"../examples/image_classifier/kitten.jpg",
+        "content-type":"application/json",
+        "validator":"image_classification",
+        "expected":[
+            {
+                "tabby":0.40864479541778564
+            },
+            {
+                "tiger_cat":0.35061511397361755
+            },
+            {
+                "Egyptian_cat":0.1260961890220642
+            },
+            {
+                "lynx":0.023326940834522247
+            },
+            {
+                "bucket":0.011855212040245533
+            }
+        ],
+        "tolerance":1
+    },
+    {
+        "url":"https://torchserve.s3.amazonaws.com/mar_files/vgg11.mar",
+        "model_name":"vgg11",
+        "worker":1,
+        "synchronous":"true",
+        "file":"../examples/image_classifier/kitten.jpg",
+        "content-type":"application/json",
+        "validator":"image_classification",
+        "expected":[
+            {
+                "tiger_cat":0.5501413345336914
+            },
+            {
+                "tabby":0.3231860101222992
+            },
+            {
+                "Egyptian_cat":0.11440037935972214
+            },
+            {
+                "lynx":0.006006305105984211
+            },
+            {
+                "tiger":0.004394155461341143
+            }
+        ],
```

Comment:
These numbers are too specific even for a test. 

Can you please add some info about testing done to confirm that the tolerance values are in the range that we expect.

Reply:
Added testing logs of the regression suite tested on the various execution environments(updated PR description).
The set tolerance value works fine on following 
1.  CPU device local execution
2. GPU  device local execution
3. CPU device, cpu docker container
4. GPU device, cpu docker container
5. GPU device, gpu docker container